packages/xml-views-validation/src/utils/deprecated-message-builder.ts

```diff
@@ -13,11 +13,10 @@ export function buildDeprecatedIssueMessage({
 }): string {
   const msgPrefix = `UI5 ${ui5Kind} ${fqn} is deprecated`;
   const sinceOptionalPart = deprecatedInfo.since
-    ? ` since: ${deprecatedInfo.since}`
-    : "";
-  const detailsOptionalPart = deprecatedInfo.text
-    ? `\n\t${deprecatedInfo.text}.`
+    ? ` since version: ${deprecatedInfo.since}`
     : "";
 
-  return `${msgPrefix}${sinceOptionalPart}.${detailsOptionalPart}`;
+  // we are ignoring the deprecated description because there is no way to render
```

Comment:
Please return the description, it has value even without the markdown

Reply:
It does not always have value, Lets consult with UI5/Product, examples:
- ![image](https://user-images.githubusercontent.com/4233843/81660638-af1f3880-9443-11ea-8021-1e8bb8843ac7.png)

- ![image](https://user-images.githubusercontent.com/4233843/81660861-f3aad400-9443-11ea-8f2e-9802952c3ec6.png)

- ![image](https://user-images.githubusercontent.com/4233843/81661009-2ce34400-9444-11ea-8be6-459f4e42234c.png)